java/vector/src/main/codegen/data/ValueVectorTypes.tdd

```diff
@@ -73,26 +73,10 @@
         { class: "UInt8" },
         { class: "Float8",           javaType: "double", boxedType: "Double", fields: [{name: "value", type: "double"}] },
         { class: "DateMilli",        javaType: "long",                      friendlyType: "LocalDateTime" },
-        { class: "TimeStampSec",     javaType: "long",   boxedType: "Long", friendlyType: "LocalDateTime" },
-        { class: "TimeStampMilli",   javaType: "long",   boxedType: "Long", friendlyType: "LocalDateTime" },
-        { class: "TimeStampMicro",   javaType: "long",   boxedType: "Long", friendlyType: "LocalDateTime" },
-        { class: "TimeStampNano",    javaType: "long",   boxedType: "Long", friendlyType: "LocalDateTime" },
-        { class: "TimeStampSecTZ", javaType: "long",   boxedType: "Long",
-                                     typeParams: [ {name: "timezone", type: "String"} ],
-                                     arrowType: "org.apache.arrow.vector.types.pojo.ArrowType.Timestamp",
-                                     arrowTypeConstructorParams: ["org.apache.arrow.vector.types.TimeUnit.SECOND", "timezone"] },
-        { class: "TimeStampMilliTZ", javaType: "long",   boxedType: "Long",
-                                     typeParams: [ {name: "timezone", type: "String"} ],
-                                     arrowType: "org.apache.arrow.vector.types.pojo.ArrowType.Timestamp",
-                                     arrowTypeConstructorParams: ["org.apache.arrow.vector.types.TimeUnit.MILLISECOND", "timezone"] },
-        { class: "TimeStampMicroTZ", javaType: "long",   boxedType: "Long",
-                                     typeParams: [ {name: "timezone", type: "String"} ],
-                                     arrowType: "org.apache.arrow.vector.types.pojo.ArrowType.Timestamp",
-                                     arrowTypeConstructorParams: ["org.apache.arrow.vector.types.TimeUnit.MICROSECOND", "timezone"] },
-        { class: "TimeStampNanoTZ", javaType: "long",   boxedType: "Long",
-                                     typeParams: [ {name: "timezone", type: "String"} ],
-                                     arrowType: "org.apache.arrow.vector.types.pojo.ArrowType.Timestamp",
-                                     arrowTypeConstructorParams: ["org.apache.arrow.vector.types.TimeUnit.NANOSECOND", "timezone"] },
+        { class: "Timestamp",        javaType: "long",   boxedType: "Long", friendlyType: "LocalDateTime"
```

Comment:
Is the `friendlyType` param still used any more?  If so then will `LocalDateTime` work with a timezone set?

Reply:
It is used. The way it currently works is if timezone is specified, if will just drop the timezone.

For example:
```
value = 0, timezone = null
```
returns `LocalDateTime(1970-01-01 00:00:00)`

```
value = 0, timezone = "America/New_York"
```
returns `LocalDateTime(1969-12-31 19:00:00)`